### PR TITLE
Issue #0000 feat: Added Validator for node validation

### PR DESCRIPTION
**Pull request by krgauraw**

### CODE REVIEW

File: ontology-engine/graph-core/src/main/java/org/sunbird/graph/util/NodeValidator.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class NodeValidator {
+    //TODO: GRAPH_ID can be fetched from configuration
+    private static final String GRAPH_ID = "domain";
```
Comment (maheshkumargangula) [inline ontology-engine/graph-core/src/main/java/org/sunbird/graph/util/NodeValidator.java]: this should be a parameter for validate method.

File: ontology-engine/graph-core/src/main/java/org/sunbird/graph/util/NodeValidator.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    public static List<Map<String, Object>> validate(List<String> identifiers) {
```
Comment (maheshkumargangula) [inline ontology-engine/graph-core/src/main/java/org/sunbird/graph/util/NodeValidator.java]: return Node List

File: ontology-engine/graph-core/src/main/java/org/sunbird/graph/util/NodeValidator.java (hunk 1 above)
Comment (krgauraw) [reply ontology-engine/graph-core/src/main/java/org/sunbird/graph/util/NodeValidator.java]: done

File: ontology-engine/graph-core/src/main/java/org/sunbird/graph/util/NodeValidator.java (hunk 2 above)
Comment (krgauraw) [reply ontology-engine/graph-core/src/main/java/org/sunbird/graph/util/NodeValidator.java]: done

**Pull request merged**